ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb

```diff
@@ -592,16 +592,16 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "image_classif_pipeline.json",
</pre><p>Is 'classif' a typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
well, it was meant to be just an abbreviation of 'classification' to make the file name shorter :) but if it seems weird maybe I should change it.